### ISSUE TITLE
Screen reader support

### ISSUE THREAD

**Issue by benwiley4000** [benwiley4000/cassette#399]
Can a blind person read the UI easily? (only applicable to the default player implementation). Buttons should ideally have `aria-label` attributes explaining their function. Note that these should be swappable with other translations ideally.. (think about that for #247).

Spun out from #261.
